ooniprobe/ooniprobe.entitlements

```diff
@@ -4,5 +4,9 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:run.ooni.org</string>
```

Comment:
Isn't it `run.ooni.io`?

Reply:
It is, are you looking at the last commit?